build.gradle

```diff
@@ -53,15 +53,19 @@ dependencies {
     annotationProcessor 'org.projectlombok:lombok:1.18.34'
     implementation 'org.springframework.boot:spring-boot-starter-validation:3.3.4'
 
-    // TEST
```

Comment:
<img width="500" height="500" alt="image" src="https://github.com/user-attachments/assets/b27e9fc7-ba67-48c6-b5a1-c1daa3d2682a" />
Please pay attention what you commit. That's... not pretty at all. Your repos should be clean